doc/release-process.md

```diff
@@ -276,3 +276,23 @@ Note: check that SHA256SUMS itself doesn't end up in SHA256SUMS, which is a spur
   - Create a [new GitHub release](https://github.com/dashpay/dash/releases/new) with a link to the archived release notes.
 
   - Celebrate
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
+Note that all values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
+
+To calculate `m_assumed_blockchain_size`:
+- For `mainnet` -> Take the size of the Bitcoin data directory, excluding `/regtest` and `/testnet3` directories.
```

Comment:
Dashify (and squash into original commit)

Reply:
Got it!!! Thank you!!!!